source/api/plugins/browser-launch-api.md

```diff
@@ -49,20 +49,29 @@ The returned `launchOptions` object will become the new launch options for the b
 
 ### Modify browser launch arguments:
 
+Here are args available for the currently supported browsers:
+
+* {% url 'Chromium-based browsers' "https://peter.sh/experiments/chromium-command-line-switches/" %}
+* {% url 'Firefox' "https://developer.mozilla.org/docs/Mozilla/Command_Line_Options" %}
+
 ```js
 module.exports = (on, config) => {
   on('before:browser:launch', (browser = {}, launchOptions) => {
+    // `args` is an array of all the arguments that will
+    // be passed to browsers when it launches
+    console.log(launchOptions.args) // print all current args
+
     if (browser.family === 'chromium' && browser.name !== 'electron') {
-      // `args` is an array of all the arguments that will
-      //  be passed to Chromium-based browsers when it launchers
-      launchOptions.args.push('--start-fullscreen')
+      // auto open devtools
+      launchOptions.args.push('--auto-open-devtools-for-tabs')
 
       // whatever you return here becomes the launchOptions
       return launchOptions
     }
 
     if (browser.family === 'firefox') {
-      launchOptions.args.push('-some-cli-argument')
+      // auto open devtools
+      launchOptions.args.push('-devtools')
 
```

Comment:
@Bkucera Is there any actual usecase for `launchOptions.args` in Electron? Mostly everything I could see pushing in is now in `preferences`

Reply:
there is no use case for `args` in electron. we just ignore them

Reply:
I added a note saying this to top of browser launch args section.